utils.py

```diff
@@ -4,6 +4,7 @@
 from os.path import isfile, join
 import zipfile
 import json
+import pathlib
 
 
```

Comment:
InstalledSoftware: remove the main function

Reply:
Didn't understand what do you mean by removing the main function,
you meant to make it instead, a function which print the fields of a key ?